package.json

```diff
@@ -128,7 +128,6 @@
     "@types/svg4everybody": "^2.1.2",
     "@typescript-eslint/eslint-plugin": "^5.15.0",
     "@typescript-eslint/parser": "^5.15.0",
-    "autoprefixer": "^10.4.2",
```

Comment:
Not sure if this is needed anywhere but looked like it wasn't being used and was not a peerdependency of anything